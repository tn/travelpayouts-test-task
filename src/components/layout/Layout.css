.layout {
    display: grid;
    grid-template-columns: 80px auto;
    grid-template-rows: 80px;
    grid-column-gap: 0px;
    grid-row-gap: 0px;
    justify-items: stretch;
    align-items: stretch;
}
