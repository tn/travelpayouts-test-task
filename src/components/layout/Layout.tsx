import * as React from 'react'
import './Layout.css'

const Layout: React.SFC = ({ children }) => (
  <div className='layout'>
    {children}
  </div>
)

export default Layout
