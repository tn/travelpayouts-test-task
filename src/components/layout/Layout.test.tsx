import * as React from 'react'
import * as Adapter from 'enzyme-adapter-react-16'
import { configure } from 'enzyme'
import * as renderer from 'react-test-renderer'
import Layout from './Layout'

configure({ adapter: new Adapter() })

describe('Layout', () => {
  it('should be rendered', () => {
    const wrapper = renderer.create(<Layout>Layout</Layout>).toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
