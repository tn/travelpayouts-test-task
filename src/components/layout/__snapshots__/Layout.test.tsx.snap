// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should be rendered 1`] = `
<div
  className="layout"
>
  Layout
</div>
`;
